src/components/Communicator/CommunicatorDialog/CommunicatorDialog.messages.js

```diff
@@ -314,5 +314,37 @@ export default defineMessages({
       'cboard.components.CommunicatorDialog.walkthroughAllMyBoardsDeleteBoard',
     defaultMessage:
       'Delete your board from the database, and it cannot be recovered.'
+  },
+  boardReport: {
+    id: 'cboard.components.CommunicatorDialog.boardReport',
+    defaultMessage: 'Report a Board'
+  },
+  boardReportContentSubtitle: {
+    id: 'cboard.components.CommunicatorDialog.boardReportContentSubtitle',
+    defaultMessage: 'Are you sure that you want to report this board?'
```

Comment:
Use the following message `Please let us know if you consider that this board includes sexual, violent, hateful, or abusive content`